src/account/accountsReducer.js

```diff
@@ -151,6 +151,20 @@ export default (state: AccountsState = initialState, action: Action): AccountsSt
             zulipFeatureLevel: zulip_feature_level,
           });
         }
+        case EventTypes.realm: {
```

Comment:
nit:
```suggestion
        }

        case EventTypes.realm: {
```

and also between the end of this case and the default case

(Arguably there should have been a blank line already separating the existing case from the default case. But perhaps it didn't seem as needed when there was just one non-default case.)